AudioKit/Platforms/Common/CsoundObj.m

```diff
@@ -142,11 +172,15 @@ - (void)record:(NSString *)csdFilePath toURL:(NSURL *)outputURL
 
 - (void)record:(NSString *)csdFilePath toFile:(NSString *)outputFile
 {
+#ifdef TRAVIS_CI
+    [self performSelectorOnMainThread:@selector(runCsoundToDisk:) withObject:@[csdFilePath, outputFile] waitUntilDone:YES];
```

Comment:
Again here, why go back to the old way of performing the selector on a thread instead of what we have with `NSThread`? It's essentially the same but gives us some additional control on the thread to terminate it more cleanly.